src/Management/test/Endpoint.Test/Actuators/Loggers/TestLoggerProvider.cs

```diff
@@ -8,34 +8,34 @@
 
 namespace Steeltoe.Management.Endpoint.Test.Actuators.Loggers;
 
-internal sealed class TestLogProvider : IDynamicLoggerProvider
+internal sealed class TestLoggerProvider : IDynamicLoggerProvider
 {
-    public string? Category { get; private set; }
-    public LogLevel MinLevel { get; private set; }
-    public bool GetLoggerConfigurationsCalled { get; private set; }
+    public bool HasCalledGetLogLevels { get; private set; }
+    public string? SetCategory { get; private set; }
+    public LogLevel SetMinLevel { get; private set; }
```

Comment:
Should this be `SetMinimumLevel` and (later in the file) `minimumLevel`?

Reply:
I don't think so. It would be inconsistent with the parameter coming from the `IDynamicLoggerProvider` interface:
```c#
public void SetLogLevel(string categoryName, LogLevel? minLevel)
{
    SetCategory = categoryName;
    SetMinLevel = minLevel ?? LogLevel.None;
}
```

Reply:
I did make the property nullable, though. Setting it to `None` is simply wrong.